packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -27,6 +27,8 @@ const isRegularObject = (value: unknown): boolean => {
 
 /**
  * Inspect the given error for common "safe" props and return them.
+ *
+ * @internal
```

Comment:
In case an external consumer was using one of these APIs, do I need to do anything to alert them?  e.g. mark it as `@deprecated` in the upcoming major release?

Not sure when we are formalizing this `@internal` tag but I've heard it's coming, I think?

cc @Josmithr

Reply:
There are plans to stop including anything marked `@internal` in our type definition files for consumers, yeah, so deprecating them now would be a good idea under the assumption that they will disappear for external consumers at some point. I'd just add `@deprecated This is an internal API not intended for external consumers and will be removed from our public API surface in a future release.` to everything you marked `@internal` and update the changeset along the same lines, "[types] are being deprecated for external consumers in preparation to eventually remove them from our public API surface". I don't think the release notes need to clarify that they're being marked `@internal` or why.

Reply:
Related design doc for reference (Microsoft only): https://microsoft.sharepoint.com/:w:/t/FluidFramework/EdNaCTjk6sNBggpWR0ARCbgB5dX6SOPQuv657tmn9DqIyA?e=dUsLGJ

Reply:
Note that if we mark the items as deprecated, we will have to disable a large number of linter errors around their use internally in the repo. I agree with @alexvy86 that this is probably the right call, but it does come with a cost.

Reply:
Great info!  Given the 11th-hour moment before internal.7.0 and lack of usage of these AFAIK / lack of urgency to truly lock this down in particular, not going to make any changes to this PR.  I'm assuming this kind of thing will happen as part of the general effort, in time.